### PR TITLE
Closes #142 142_pkg_versions: add minimum package versions

### LINKED ISSUE #142: Feature Request:  Declare minimum package version on Imports
### Feature Idea

Declare minimum package version on Imports.  We just set this up in admiral-core.  Thought it might be worthwhile for adminralonco and admiraldev to have these as well.

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by bundfussr**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -38,13 +38,13 @@ Roxygen: list(markdown = TRUE)
...
-    dplyr,
-    lubridate,
-    magrittr,
-    rlang,
-    tidyselect
+    dplyr (>= 0.8.4),
+    lubridate (>= 1.7.4),
+    magrittr (>= 1.5),
+    rlang (>= 0.4.4),
+    tidyselect (>= 1.0.0)
```
Comment (sgorm123) [inline DESCRIPTION:47]: I am sure these are all fine @bundfussr, but for completeness, where are the version minimums that we need documented? As I see there are many later versions of each of these packages on CRAN.

Comment (bundfussr) [reply DESCRIPTION:47]: @sgorm123 , I am not sure what you mean by "documented".

**Review by rossfarrugia — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (rossfarrugia) [reply DESCRIPTION:47]: i would always look to the description file for these, so for me this PR itself covers this, and no further documentation needed.

Comment (sgorm123) [reply DESCRIPTION:47]: As discussed @bundfussr, I just wondered if there was maybe documentation/process in which minimum versions are reviewed and updated.  If the PR itself covers this, as Ross says, then no problem.  Thank you both for responding.

**Pull request merged**